cirq/google/calibration/phased_fsim.py

```diff
@@ -33,6 +46,14 @@
     from cirq.protocols import json_serializable_dataclass
 
 
+T = TypeVar('T')
+
+
+# Workaround for: https://github.com/python/mypy/issues/5858
+def lru_cache_typesafe(func: Callable[..., T]) -> T:
```

Comment:
Do we have an issue where we can record all the places where mypy makes things worse? @balopat